tools/commentMonitor/client.go

```diff
@@ -31,6 +31,7 @@ type commentMonitorClient struct {
 	events             []webhookEvent
 	prefixes           []commandPrefix
 	prefixHelpTemplate string
+	prefixVerifyUser   bool
```

Comment:
Why are these called "prefixHelpTemplate" and "prefixVerifyUser", why not just helpTemplate and verifyUser?